### DISCUSSION TITLE
Uso de um sistema online para votação da diretoria da APyB em 2023

### DISCUSSION THREAD

**Discussion by gabubellon** [apyb/comunidade#90] [Geral]
Olá a todos

Esse ano iremos ter novamente votação para a diretoria da APyB e gostaria de sugerir para realizar a mesma de maneira virtual.

Sei que a própria PSF realiza suas votação de diretoria utilizando o [Helios](https://github.com/benadida/helios-server).

Podemos seguir a mesma, visto que  com isso  temos uma acessibilidade de votação maior, podendo até despertar o interesse de se aproximar da APyB pessoas que só entendem que precisam estar no evento anual para o mesmo.

Gostaria de saber a opinião da comunidade e contém com minha ajuda para tirar isso do papel

cc @apyb/diretoria @apyb/conselho-deliberativo

**Comment by rougeth** [comment]
Eleições da diretoria e conselhos precisam ser feitas, exclusivamente, durante a assembleia geral:

```
Art. 11º - Compete privativamente à assembléia geral:

- eleger os Diretores e Conselheiros;
- destituir os Diretores e Conselheiros;
- alterar o estatuto.
- aprovar as contas e relatório constante no item "a" art 29.º deste Estatuto.
```

https://github.com/apyb/estatuto#cap%C3%ADtulo-iii---da-assembl%C3%A9ia

Nós estamos discutindo sobre como fazer a assembleia esse ano. Por um lado, deixar de fazer o encontro presencialmente, durante a Python Brasil, acabaríamos perdendo muito do engajamento que o evento trás. Basta lembrar que a maioria das cidades sedes e chapas surgiram no decorrer do evento.

Dito isso, também queremos deixar o encontro mais acessível para quem não poderá viajar até Caxias, então estamos considerando fazer a assembleia de maneira híbrida esse ano.

**Comment by deboraazevedo** [comment]
A PSF de fato utilizava o Helios como plataforma de votação na eleição da diretoria, mas isso foi mudado esse ano e a plataforma utilizada foi o Opa Vote. É uma plataforma paga mas se considerou um bom investimento.

**Comment by tiidadavena** [comment]
> A PSF de fato utilizava o Helios como plataforma de votação na eleição da diretoria, mas isso foi mudado esse ano e a plataforma utilizada foi o Opa Vote. É uma plataforma paga mas se considerou um bom investimento.

Não conhecia mas achei muito interessante a ideia de personalizar o preço da plataforma via quantidade de votantes e de candidaturas: https://www.opavote.com/pricing
